share/parse_sparse_shares_test.go

```diff
@@ -143,15 +145,21 @@ func Test_parseSparseSharesWithNamespacedPadding(t *testing.T) {
 }
 
 func Test_parseShareVersionOne(t *testing.T) {
-	v1blob, err := NewV1Blob(MustNewV0Namespace(bytes.Repeat([]byte{1}, NamespaceVersionZeroIDSize)), []byte("data"), bytes.Repeat([]byte{1}, SignerSize))
```

Comment:
[note] this didn't previously repro the error because the blob data used was only `[]byte("data")` which fits in one share.